core/services/ardupilot_manager/mavlink_proxy/MAVLinkRouter.py

```diff
@@ -55,8 +61,8 @@ def binary_name() -> str:
     def _validate_endpoint(endpoint: Endpoint) -> bool:
         return endpoint.connection_type in [
             EndpointType.UDPServer,
-            EndpointType.UDPClient,
-            EndpointType.Serial,
```

Comment:
Why UDPServer is here and Serial not if both are valid master endpoints ?

Reply:
This validation is for output endpoints, not master.

Reply:
That's my point, UDPClient is the endpoint and not UDPServer, UDPServer is master.

Reply:
True. I responded that one before the other conversation hehe